### DISCUSSION TITLE
Input Script Validation

### DISCUSSION THREAD

**Discussion by AnuragKr** [deepmodeling/deepmd-kit#2173] [Q&A] [answered]
Hi,
Objective - To draw this graph.
![dp_compress_network_structure](https://user-images.githubusercontent.com/10074710/206382265-7c87ee5d-d3c2-47b6-ae62-7630de00a7d5.png)

Input Script

```json
"model": {
        "type_map":     ["O", "H"],
        "type_embedding":{
                "neuron":[2,4,8],
                "seed":1
        },
        "descriptor" :{
            "type":             "se_e2_a",
            "sel":              [46, 92],
            "rcut_smth":        0.50,
            "rcut":             6.00,
            "neuron":           [25, 50, 100],
            "resnet_dt":        false,
            "axis_neuron":      16,
            "seed":             1,
            "_comment":         " that's all"
        },
        "fitting_net" : {
            "neuron":           [15, 15, 15],
            "resnet_dt":        true,
            "seed":             1,
            "_comment":         " that's all"
        },
        "_comment":     " that's all"
    }
```

Does descriptor neuron structure depend on embedding net output?  Or Can I change the embedding net neuron structure keeping the descriptor neuron structure the same?

**Comment by AnuragKr** [comment]
@njzjz  Please help me in figuring out the above doubt.

**Comment by njzjz** [comment, marked as answer]
> Does descriptor neuron structure depend on embedding net output?

For "embedding net", do you mean the type embedding? If so, they are independent.

**Reply by AnuragKr** [reply]
Yes, I meant type embedding. Thanks for the clarification.
